src/Stratis.Bitcoin.Features.Consensus.Tests/CoinViews/CoinViewTest.cs

```diff
@@ -11,11 +10,17 @@ namespace Stratis.Bitcoin.Features.Consensus.Tests.CoinViews
 {
     public class CoinViewTest
     {
-        private TestCoinView coinView;
+        private ICoinView coinView;
 
         public CoinViewTest()
         {
-            this.coinView = new TestCoinView();
+            var coinViewMock = new Mock<ICoinView>();
+            var fetchCoinResponse = new FetchCoinsResponse(new UnspentOutputs[0], new uint256(987263876253));
+            coinViewMock.Setup(c => c.FetchCoinsAsync(It.IsAny<uint256[]>(), It.IsAny<CancellationToken>()))
+                .ReturnsAsync(fetchCoinResponse);
+            coinViewMock.Setup(c => c.GetTipHashAsync(It.IsAny<CancellationToken>()))
+                .ReturnsAsync(fetchCoinResponse.BlockHash);
+            this.coinView = coinViewMock.Object;
         }
```

Comment:
Maybe move this into the actual test, add some line spacing, and make uint256(987263876253) a variable ?

Reply:
Test refactoring will be done in a separate PR as per my PR summary.